dotcom-rendering/src/lib/useIsSignedIn.ts

```diff
@@ -0,0 +1,35 @@
+import { getCookie } from '@guardian/libs';
+import { useEffect, useState } from 'react';
+
+async function doOktaMethodForSignIn() {
```

Comment:
because of its name, it _feels_ like this method is doing the sign in, which I don't think it is. I wonder if we could rename it to something like 

- `getSignInStatusWithOkta`
- `getIsSignedInWithOkta`
- another suggestion

Reply:
It's maybe a bit lengthy, but I'm thinking `getIsSignedInStatusWithOkta` (since it returns something that we use to set the state of `isSignedIn`)